src/configs/getAndroidConfig.ts

```diff
@@ -40,15 +46,72 @@ export default function getAndroidConfig(theme: Theme): Platform {
         destination: `${_.upperFirst(_.camelCase(theme))}DesignTokens.kt`,
         className: _.upperFirst(_.camelCase(theme)) + "DesignTokens",
         packageName: "io.element.android.compound",
+        filter: function(token: TransformedToken) {
+          return !isFontType(token) && isCoreColor.matcher(token);
```

Comment:
I believe we should be fine by only using the `isCoreColor` filter here.
If it's a color, it can't be a font

Reply:
Fixed by 0a1d0e18f320aeb5ec8f65714cdd8af9855d8581.